app/controllers/EventReportController.scala

```diff
@@ -161,9 +161,10 @@ class EventReportController @Inject()(
         if (notFoundPSTR.contains(pstr) || pstr.matches(perfTestPstrPattern))
           Future.successful(NotFound(invalidPstrResponse))
         else {
-          val jsValue = jsonUtils.readJsonIfFileFound(s"$path/${pstr}_$eventType.json")
-            .getOrElse(defaultGetEvent1832())
-          Future.successful(Ok(jsValue))
+          jsonUtils.readJsonIfFileFound(s"$path/${pstr}_$eventType.json") match {
+            case Some(jsValue) => Future.successful(Ok(jsValue))
+            case None => Future.successful(NotFound(invalidPstrResponse))
```

Comment:
I'd change the invalid pstr response text to something more appropriate. If you look in yaml files for APIs you will find something indicating that there is nothing found - so use that